paper-server/patches/sources/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -60,6 +60,36 @@
              }
          }
  
+@@ -394,10 +_,10 @@
+             boolean flag = this instanceof Player;
+             if (this.isInWall()) {
+                 this.hurtServer(serverLevel1, this.damageSources().inWall(), 1.0F);
+-            } else if (flag && !serverLevel1.getWorldBorder().isWithinBounds(this.getBoundingBox())) {
+-                double d = serverLevel1.getWorldBorder().getDistanceToBorder(this) + serverLevel1.getWorldBorder().getDamageSafeZone();
++            } else if (flag && !this.getWorldBorder().isWithinBounds(this.getBoundingBox())) { // Paper - use the first WorldBorder available
++                double d = this.getWorldBorder().getDistanceToBorder(this) + serverLevel1.getWorldBorder().getDamageSafeZone(); // Paper - use the first WorldBorder available
```

Comment:
Very nitpicky
But I don't believe "use first world border available" really describes the changes

Maybe smth like "use players worldborder, if present"